apps/desktop/src/lib/branch/SeriesHeader.svelte

```diff
@@ -192,6 +192,9 @@
 					await updateStatusAndChecks();
 				})
 				.catch((err) => {
+					if (err.message.includes('Cannot change the base branch of a closed pull request'))
+						return;
+
```

Comment:
Actually I'm thinking both this the swallowing in this PR and the swallowing of that error you're pointing out should probably not be required any more once we merge this new ["update pr base branch at the end of 'merge' event handler instead of in $effect"](https://github.com/gitbutlerapp/gitbutler/pull/5797) PR.